pkg/utils/miniojob/minioJob.go

```diff
@@ -155,6 +207,22 @@ func KeyFormat(key string, format string) FieldsFunc {
 	}
 }
 
+// KeepKey - match key and output the key
+func KeepKey(key string) FieldsFunc {
```

Comment:
Isn't it strange to call a function`KeepKey`, while it actually removes the key and outputs it?

Reply:
Hmm. KeepKey to command. Like input enable, so command print enable.

Reply:
@harshavardhana Any names for this?